x/posts/client/cli/tx.go

```diff
@@ -67,11 +80,39 @@ func GetCmdCreatePost(cdc *codec.Codec) *cobra.Command {
 				return err
 			}
 
-			msg := types.NewMsgCreatePost(args[1], parentID, allowsComments, args[0], map[string]string{}, from, time.Now().UTC())
+			msg := types.NewMsgCreatePost(
+				args[1],
+				parentID,
+				allowsComments,
+				args[0],
+				map[string]string{},
+				from,
+				time.Now().UTC(),
+				nil,
+			)
+
+			// If there are some medias
+			if len(args) > 3 {
+				var medias types.PostMedias
+				// Read each media and add it to the medias if valid
+				for i := 3; i < len(args); i++ {
+					arg := strings.Split(args[i], ",")
+					if len(arg) != 2 {
+						return fmt.Errorf("if medias are specified, they shouldn't have empty fields, please use the " +
+							"--help flag to know more")
```

Comment:
Probably we can have this on the same line of the first part of the message, even if it exceeds the limit